sample/src/main/java/com/chuckerteam/chucker/sample/HttpBinClient.kt

```diff
@@ -145,4 +149,32 @@ class HttpBinClient(
             }
         )
     }
+
+    private fun getProtoResponse() {
+        val pokemon = Pokemon("Pikachu", level = 99)
+        val body = pokemon.encodeByteString().toRequestBody("application/protobuf".toMediaType())
+        val request = Request.Builder()
+            .url("https://postman-echo.com/post")
+            .post(body)
+            .build()
+        httpClient.newCall(request).enqueue(
+            object : okhttp3.Callback {
+                override fun onFailure(call: okhttp3.Call, e: IOException) = Unit
+
+                override fun onResponse(call: okhttp3.Call, response: okhttp3.Response) {
+                    response.body?.source()?.use { it.readByteString() }
+                }
+            }
+        )
+    }
+
+    private class PokemonProtoBodyDecoder : BodyDecoder {
+        override fun decodeRequest(request: Request, body: ByteString): String? {
+            return if (request.url.host.contains("postman", ignoreCase = true)) {
+                Pokemon.ADAPTER.decode(body).toString()
+            } else null
+        }
+
+        override fun decodeResponse(response: okhttp3.Response, body: ByteString): String? = null
+    }
```

Comment:
Please move this to its own file. Users could take this as inspiration and it's better if it's more prominent.